pkg/service/pipeline.go

```diff
@@ -1064,36 +1103,44 @@ func (s *service) preTriggerPipeline(ctx context.Context, ns resource.Namespace,
 					if err != nil {
 						return err
 					}
+					uploadingPipelineData[idx][k] = v.GetStringValue()
```

Comment:
@chuang8511 
The URL might still be a dataURI (base64). If the user continues using dataURI or base64 in the input, we should save it to our blob storage and replace it with the corresponding blob URL. Could you check if this change can be implemented quickly? I guess we can modify lines L868–L907 to handle this.

Reply:
> The URL might still be a dataURI (base64).

Do you mean when the users call API to trigger pipeline but not from Console?

Reply:
Yes, that’s one of the possibilities.

Reply:
fixed in [this commit](https://github.com/instill-ai/pipeline-backend/pull/904/commits/8b5decb2e659173b0e044753eee49f17c6751670).